src/templates/zombienet/zombienet.config.toml

```diff
@@ -0,0 +1,16 @@
+[settings]
+timeout = 1000
```

Comment:
should we include something similar in the `polkadot.toml` file as well?

Reply:
yes, I think the `polkadot.toml` is a bit redundant with the default `zombienet.config.toml`, I will probably removed it